Brokerages/Binance/BinanceRestApiClient.cs

```diff
@@ -317,6 +317,11 @@ public IEnumerable<Messages.Kline> GetHistory(Data.HistoryRequest request)
                 var klines = JsonConvert.DeserializeObject<object[][]>(response.Content)
                     .Select(entries => new Messages.Kline(entries))
                     .ToList();
+
+                var windowStartTime = (new DateTime(1970, 1, 1)).AddMilliseconds(klines.First().OpenTime);
+                var windowEndTime = (new DateTime(1970, 1, 1)).AddMilliseconds(klines.Last().OpenTime + resolutionInMs);
+                Log.Debug($"Received [{symbol}] data for timeperiod from {windowStartTime.ToStringInvariant()} to {windowEndTime.ToStringInvariant()}..");
```

Comment:
Should move this inside the next `if (klines.Count > 0)`, since `klines` might be empty? I bet we have helper methods for this `(new DateTime(1970, 1, 1)).AddMilliseconds()`